### PR TITLE
Add windows support

### PR DESCRIPTION
**Pull request by libre-man**

### CODE REVIEW

File: lib/codegra-atom.js [hunk 1]
```diff
@@ -157,31 +173,24 @@ function getFileComments(filePath) {
...
 
 function addFileComments(filePath, newComments) {
-	let comments = [];
-	getFileComments(filePath).forEach((comment) =>
-		comments[comment.location.position.start.row] = comment);
-	newComments.forEach((comment) =>
-		comments[comment.location.position.start.row] = comment);
+	let comments = [].concat(getFileComments(filePath)).concat(newComments);
 	// Condense sparse array
 	comments = comments.filter((x) => true);
```
Comment (olmokramer) [inline lib/codegra-atom.js]: We don't need this anymore right?

File: lib/codegra-atom.js [hunk 2]
```diff
@@ -157,31 +173,24 @@ function getFileComments(filePath) {
...
 	linter.setMessages(filePath, comments);
 }
 
 function deleteFileComments(filePath, lineNrs) {
-	let comments = [];
-	getFileComments(filePath).forEach((comment) =>
-		comments[comment.location.position.start.row] = comment);
-	lineNrs.forEach((line) => delete comments[line]);
+	let comments = [].concat(getFileComments(filePath));
```
Comment (olmokramer) [inline lib/codegra-atom.js]: The `[].concat` here is useless.

File: lib/codegra-atom.js [hunk 3]
```diff
@@ -157,31 +173,24 @@ function getFileComments(filePath) {
...
 	// Condense sparse array
-	comments = comments.filter((x) => true);
+	comments = comments.filter((msg) => lineNrs.indexOf(msg.location.position.start.row) < 0);
```
Comment (olmokramer) [inline lib/codegra-atom.js:191]: On line 193 we use `.row[0]`, shouldn't we do that here as well?

File: lib/codegra-atom.js (hunk 1 above)
Comment (libre-man) [reply lib/codegra-atom.js]: Correct.

File: lib/codegra-atom.js (hunk 2 above)
Comment (libre-man) [reply lib/codegra-atom.js]: Agree.

File: lib/codegra-atom.js (hunk 3 above)
Comment (libre-man) [reply lib/codegra-atom.js:191]: Yeah we should, good catch!

Comment (olmokramer) [reply lib/codegra-atom.js:191]: Apparently we shouldn't... Doesn't work for me when I add it.

**Pull request merged**